dist/media-query-provider.js

```diff
@@ -36,16 +36,16 @@ var defaultQueries = {
 var MediaQueryProvider = function (_React$Component) {
   _inherits(MediaQueryProvider, _React$Component);
 
-  function MediaQueryProvider() {
+  function MediaQueryProvider(props) {
```

Comment:
any reason for including built files in the repo?

Reply:
Can fix this in a future PR - should ideally add things like `in-publish` and `version-changelog`